CP01CNPM/CP01CNPM_R00003_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00003/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP01CNPM-RII01-02-ADCPTG010,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP01CNPM/R00003/instruments/ADCP_sn*/*000.000,CP01CNPM-RII01-02-ADCPTG010,recovered_inst,Available,

```

Comment:
Why does this one have *000.000, instead of just *.000 like in CP01CNPM_R00002?

Reply:
I changed this to *.000 to match CP01CNPM_R00002